zadaci/sa-kolokvijuma/2020/PSI/T5/G7/zadatak.rst

```diff
@@ -0,0 +1,42 @@
+Paketi su opisani na sledeći način:
+
+* sadržaj (jedna reč, maksimalno 30 karaktera)
+* masa (realna vrednost, u kilogramima)
+* lomljivo (jedna reč, "DA", ili "NE", maksimalno 2 karaktera)
+  
+Paketi se šalju putem kurirske službe, koja naplaćuje usluge slanja paketa u odnosu na njegovu masu.
+Učitati pakete u jednostruko spregnutu listu (dodati novi element na kraj liste) i uraditi sledeće:
+
+a) Izračunati cenu dostave paketa na osnovu sledećeg cenovnika:
+
++-----------+------+
+| Masa [kg] | Cena |
++===========+======+
+| 0-5       | 200  |
++-----------+------+
+| 5-10      | 400  |
++-----------+------+
+
+Ukoliko paket ima više od 10kg, dodatna masa paketa se naplaćuje pomnožena sa vrednošću argumenta komandne linije, koji predstavlja cenu po kilogramu.
+
+b) Na standardni izlaz (ekran Terminala) ispisati sledeće informacije:
+
+* sadržaj (ispisano sa ``%-10s`` format specifikatorom)
```

Comment:
Please make these a nested list

Reply:
It converted to a block-quote with an embedded list now :/ We actually want a nested list (`*`), within a list (`b)`) - align the `*` to the text of its parent list.

BTW, this is why my assignment don't have sub-assignments under a, b, c, ... :)

Reply:
zbirka.pdf:

![image](https://user-images.githubusercontent.com/12072588/116156773-3664cb00-a6ec-11eb-9c13-66e69156bd78.png)

zbirka.html:

![image](https://user-images.githubusercontent.com/12072588/116156828-4b415e80-a6ec-11eb-8871-0eb9a05c40c9.png)